engine/src/test/java/io/camunda/zeebe/engine/processing/deployment/DeploymentRejectionTest.java

```diff
@@ -220,4 +211,43 @@ public void shouldRejectDeploymentIfResourceIsTooLarge() {
         .hasRejectionType(RejectionType.EXCEEDED_BATCH_RECORD_SIZE)
         .hasRejectionReason("");
   }
+
+  // https://github.com/camunda/zeebe/issues/8026
+  @Test
+  public void shouldRejectDeploymentOfSAXException() {
+    // given
+    final String resource = "/processes/saxexception-error-8026.bpmn";
+
+    // when
+    final Record<DeploymentRecordValue> deploymentRejection =
+        ENGINE.deployment().withXmlClasspathResource(resource).expectRejection().deploy();
+
+    // then
+    Assertions.assertThat(deploymentRejection)
+        .hasKey(ExecuteCommandResponseDecoder.keyNullValue())
+        .hasRecordType(RecordType.COMMAND_REJECTION)
+        .hasIntent(DeploymentIntent.CREATE)
+        .hasRejectionType(RejectionType.INVALID_ARGUMENT);
+
+    /*
+     The detail rejection reason is an i18n SAXException, so here use the matches instead.
+
+     In Chinese
+     "Expected to deploy new resources, but encountered the following errors:
+     '/processes/saxexception-error-8026.bpmn': SAXException Error: URI=null Line=33:
+     cvc-complex-type.3.2.2: 元素 'bpmndi:BPMNPlane' 中不允许出现属性 'stroke'。"
+
+     In English
+     "Expected to deploy new resources, but encountered the following errors:
+     '/processes/saxexception-error-8026.bpmn': SAXException Error: URI=null Line=33:
+     cvc-complex-type.3.2.2: Attribute 'stroke' is not allowed to appear in element
+     'bpmndi:BPMNPlane'.
+    */
+
+    assertThat(deploymentRejection.getRejectionReason())
+        .describedAs("rejection should contain enough detail rejection reason")
+        .contains("cvc-complex-type.3.2.2")
+        .contains("stroke")
+        .contains("bpmndi:BPMNPlane");
+  }
```

Comment:
🔧 Let's also verify that it adds the name of the bpmn file

Reply:
Done.

Hi @remcowesterhoud. Please check this out. ❤️